instagram/src/App.js

```diff
@@ -1,25 +1,75 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
 
+import PostContainer from './components/PostContainer/PostContainer.js'
+import SearchBar from './components/SearchBar/SearchBar.js'
+import dummyData from './dummy-data';
+
 class App extends Component {
+  constructor() {
+    super()
+
+    this.state = {
+      posts: [],
+      searchData: ''
+    }
+  }
+
```

Comment:
great job with everything on this file. Your code looks good, things make sense. I'd only ask that you go back to commenting more on your code again to explain what things are supposed to be doing. 

Some things are obvious like componentDidMount is initializing your dummy data and addLike is to add a like, but adding a line or two to say what's expected (addLike will increase likes on a post only) or (addLike will increase a like on a post by 1 if clicked one, or decrease it by 1 if already liked) etc.